### ISSUE TITLE
Button: keyboard focus indicator lacking affordance in Chrome

### ISSUE THREAD

**Issue by ianmcburnie** [eBay/skin#2075]
Fake button has clear focus indicator:

![Screen Shot 2023-06-02 at 9 53 19 AM](https://github.com/eBay/skin/assets/38065/c3ea1579-b11f-46fc-a47f-af06c172f754)

Real button does not:

![Screen Shot 2023-06-02 at 9 52 25 AM](https://github.com/eBay/skin/assets/38065/4d6d100c-6357-4184-9b2e-a80e9f420c43)

**Comment by saiponnada** [comment]
This will be addressed while adding the enhancement #2421
